grammars/stata-webdoc.cson

```diff
@@ -0,0 +1,32 @@
+'scopeName': 'source.webdoc.stata'
+'name': 'Stata Webdoc'
+'fileTypes': [
+  'do'
+  'dowd'
+]
+'firstLineMatch': 'webdoc init'
```

Comment:
This doesn't do what you think. This is a union among `fileTypes` and `firstLineMatch`. So any `.do` files are considered Stata Webdoc as of now.

Reply:
Ah I see. I was hoping there was some kind of "main language" identifier for `source.stata` in the package. Because as is the `firstLineMatch` doesn't add anything of value. I removed lines 4 and 7 in the new commit.